test/requirements.txt

```diff
@@ -0,0 +1,6 @@
+python3 -m venv gym-supermariobros-env
```

Comment:
The test/requirements.txt file seems like it should be a bash file - it should be appropriately named and placed in the root directory: (i.e. setup.sh).

Reply:
well i changed the requirements.txt, do I still need a bash file what would even be in a setup.sh file